code/game/objects/items/weapons/storage/fancy.dm

```diff
@@ -184,6 +185,16 @@
 /obj/item/weapon/storage/fancy/matchbox/empty
 	empty = 1
 
+//Needs to be modified, because plural is -es instead of -s
+/obj/item/weapon/storage/fancy/matchbox/examine(mob/user)

```

Comment:
This is going to make a duplicate message appear.
Don't do it. Just make a way for it to display a plural instead.